reactos/base/applications/clipbrd/lang/pl-PL.rc

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Adding this file doesn't do anything, you need to also reference it in clipbrd.rc